docker/binder/requirements_flink.txt

```diff
@@ -1,2 +1,4 @@
 -r requirements_base.txt
 apache-flink==1.12.1
+# flink requires pandas<1 see https://github.com/apache/flink/blob/release-1.12.1/flink-python/setup.py#L319-L320
```

Comment:
Not against this change, but can you move this and the change in requirements_lint.txt to another PR?